docs/customisation.md

```diff
@@ -1,25 +1,26 @@
-# Dockerfile Customisation
+# Dockerfile customization
 
-A custom Docker file might be required in certain circumstances. These include (but aren't limited to)
+It is likely, that you might need to customize your Docker file for certain purposes which include but aren't limited to:
 
-- Connecting to a properietary database (DB2, Oracle, etc.)
+- Connecting to a proprietary database (DB2, Oracle, etc.)
 - Adding a hotfix as instructed by our support team
 - Adding a custom plugin
-- etc.
 
-The process to customise a docker image for your needs is described below
+
+You can customize a docker image for your needs as described in the process below.
 
 ## XL Deploy
 
-When creating a Dockerfile with custom resources added, always remember that the owning user:group combination _must_ be `10001:0`, or else XL Deploy will not be able to read the files. 
+**Important:** When you create a Dockerfile with custom resources added, always remember that the owning user:group combination _must_ be `10001:0`, or else, XL Deploy will not be able to read the files.
 
-To start off with, create a `Dockerfile` that resembles the following. 
+**Note:** Certain JARS should be placed in specific paths only. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team.
 
-Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+**Note:** `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
 
-NOTE: `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
+To begin, create a `Dockerfile` that resembles the following configuration:
 
-```docker
+```
```

Comment:
this is incorrect, should be on beginning like ` ```docker ` as it was to use correct syntax highlight

Reply:
Understood, will change it. Thanks.